tortoise/verifying_tortoise.go

```diff
@@ -419,75 +398,65 @@ func (t *turtle) calculateExceptions(
 	if startLayer.Before(types.GetEffectiveGenesis()) {
 		startLayer = types.GetEffectiveGenesis()
 	}
-	for layerID := startLayer; !layerID.After(t.Last); layerID = layerID.Add(1) {
-		logger := logger.WithFields(log.Named("diff_layer_id", layerID))
+	for lid := startLayer; !lid.After(t.Last); lid = lid.Add(1) {
```

Comment:
Why do you want to rename `layerID` to `lid`?

Reply:
i tend to use bid and lid consistently, and was going to change everything to this format at least in tortoise. they are both short and very clear (in my opinion). if there is something specific like base block id - it is basebid, or base block layer - baselid.